graviti/manager/branch.py

```diff
@@ -55,7 +55,9 @@ class BranchManager:
     def __init__(self, dataset: "Dataset") -> None:
         self._dataset = dataset
 
-    def _generate(self, offset: int = 0, limit: int = 128) -> Generator[Branch, None, int]:
+    def _generate(
+        self, offset: Optional[int], limit: Optional[int]
```

Comment:
`_generate` is only for `LazyPagingList`, I think the offset and limit type can be just `int`